scenarios/throughput_stress.go

```
@@ -134,8 +135,17 @@ func (t *tpsExecutor) Run(ctx context.Context, info loadgen.ScenarioInfo) error
 					NexusEndpoint:                nexusEndpoint,
 					SleepActivityPerPriority:     sleepActivityPerPriority,
 				})
-			// The 1 is for the final workflow run
-			t.workflowCount.Add(uint64(result.TimesContinued + result.ChildrenSpawned + 1))
```

Noticed when testing that this is actually wrong as it ignores the error.